elastic/integrations
# Add Performance config to resolve bug

- Bug


## What does this PR do?

This PR resolves the bug caused by performance data stream in mysql as configs were not getting passed.

## Checklist




## How to test this PR locally

Run the mysql instance, can also use docker container for setup
Add Mysql integration
Enable performance metrics
Data should start flowing through ES.


- Closes #3514
- Closes #2349
